README.md

```diff
@@ -47,15 +47,37 @@ The example pipeline contains 4 blocks:
 ### Installation
 
 ```bash
-$ npm install
+$ yarn
 ```
 
+##### IOS
+
+```bash
+$ cd ios
+$ pod install
+```
+
+##### Android
+
+Make sure your `JAVA_HOME`, `ANDROID_HOME`, and `ANDROID_SDK_ROOT` environment variables are all setup.
```

Comment:
Would it make sense to include an example of the ANDROID_SDK_ROOT, or how to find that path?